### ISSUE TITLE
Bug: `use-baseline` reports `linear-gradient` and `radial-gradient` as not being widely available

### ISSUE THREAD

**Issue by jimmy-zhening-luo** [eslint/css#135]
### Environment

ESLint version: 9.26.0
@eslint/css version: 0.8.0
Node version: 22.15.1
npm version: 11.4.0
Operating System: Ubuntu & Windows 11 Pro

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```js
import { defineConfig } from "eslint/config";
import css from "@eslint/css";

export default defineConfig([
	// lint CSS files
	{
		files: ["**/*.css"],
		language: "css/css",
		plugins: { css },
		extends: ["css/recommended"],
	},

	// your other configs here
]);

```
</details>

```js
body {
  background-size: 100vi 100vb;
  background-image:
    linear-gradient(180deg,
      var(--color-bg-0) 0%,
      var(--color-bg-1) 15%,
      var(--color-bg-2) 50%) /* should be OK, but errors */;
    radial-gradient(50% 50% at 50% 50%,
      rgba(255, 255, 255, 0.75) 0%,
      rgba(255, 255, 255, 0) 100%) /* should be OK, but errors */,
}

```

### What did you expect to happen?

I expected the above code to not produce any error.

## Reasoning

[`linear-gradient`](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient) and [`radial-gradient`](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient) are considered widely-available by `mdn`, with few exceptions noted [here](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient#browser_compatibility) and [here](https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient#browser_compatibility).

### What actually happened?

Received the following errors (with screenshot):

> Type 'linear-gradient' is not a widely available baseline feature.eslint[css/use-baseline](https://github.com/eslint/css/blob/main/docs/rules/use-baseline.md)

> Type 'radial-gradient' is not a widely available baseline feature.eslint[css/use-baseline](https://github.com/eslint/css/blob/main/docs/rules/use-baseline.md)

![Image](https://github.com/user-attachments/assets/99abbd67-18ac-4cf3-9eb4-1aebfea9160e)

### Link to Minimal Reproducible Example

Please use the exact config and exact code above for minimum repo

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
Thanks for the report. It looks like this data is incorrect in the [`web-features`](https://www.npmjs.com/package/web-features) package. When we did the last update, it reset these functions to a lower baseline status:
https://github.com/eslint/css/pull/130/files#diff-79efeb98f6abe2af697f58bc91d99598dc2dea465ac51f519abce5795ef94606L597-L600

@rviscomi  can you confirm?

**Comment by nzakas** [comment]
Looks like a couple other gradient-related types also got caught in this update:
https://github.com/eslint/css/pull/130/files#diff-79efeb98f6abe2af697f58bc91d99598dc2dea465ac51f519abce5795ef94606L575-L576

**Comment by rviscomi** [comment]
Looks like a bug in the `types` regex. I have a fix in #137.

**Comment by jimmy-zhening-luo** [comment]
@rviscomi Thank you very much!
